content/docs/2.15/reference/_index.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Reference"
+weight = 2
++++
+
+Reference information for the KEDA autoscaler.
+
+- [Scaled Object specification](./scaledobject-spec)
+- [ScaledJob specification](./scaledjob-spec.md)
+- [Events]
```

Comment:
Broken link + let's clarify that this is for Kubernetes events

Reply:
Or include all events. Are there other options besides Kubernetes and cloud? Or the list too open-ended to keep a reference up to date?